common/meta/raft-store/src/grpc/grpc_action.rs

```diff
@@ -0,0 +1,209 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::convert::TryInto;
+use std::fmt::Debug;
+use std::sync::Arc;
+
+use common_exception::ErrorCode;
+use common_meta_types::CreateDatabaseReply;
+use common_meta_types::CreateDatabaseReq;
+use common_meta_types::CreateTableReply;
+use common_meta_types::CreateTableReq;
+use common_meta_types::DatabaseInfo;
+use common_meta_types::DropDatabaseReply;
+use common_meta_types::DropDatabaseReq;
+use common_meta_types::DropTableReply;
+use common_meta_types::DropTableReq;
+use common_meta_types::GetDatabaseReq;
+use common_meta_types::GetKVActionReply;
+use common_meta_types::GetTableReq;
+use common_meta_types::ListDatabaseReq;
+use common_meta_types::ListTableReq;
+use common_meta_types::MGetKVActionReply;
+use common_meta_types::MetaId;
+use common_meta_types::PrefixListReply;
+use common_meta_types::TableInfo;
+use common_meta_types::UpsertKVAction;
+use common_meta_types::UpsertKVActionReply;
+use common_meta_types::UpsertTableOptionReply;
+use common_meta_types::UpsertTableOptionReq;
+use tonic::Request;
+
+use crate::protobuf::GetReq;
+use crate::protobuf::RaftRequest;
+
+pub trait RequestFor {
+    type Reply;
+}
+
+// Action wrapper for do_action.
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, derive_more::From)]
+pub enum MetaGrpcWriteAction {
```

Comment:
```suggestion
pub enum MetaGrpcWriteReq {
```

What about use suffix `Req` for a wrapper of `**Req`s?